config/initializers/carrierwave.rb

```diff
@@ -12,6 +10,7 @@
       google_project:         'we-meditate',
       google_json_key_string: ENV['GOOGLE_CLOUD_KEYFILE'].present? ? ENV['GOOGLE_CLOUD_KEYFILE'] : nil,
     }
+    config.storage = :fog
```

Comment:
Weird, but I had to move this line:
https://github.com/carrierwaveuploader/carrierwave/issues/2023#issuecomment-252407542

Reply:
Huh, surprising that this hasn't become obvious more quickly. Considering all these breaking issues you're encountering.

Reply:
Normally this code isn't executed in dev environment (since GCLOUD_BUCKET won't be set). So perhaps some discrepancy between dev and staging/production environment. Have yet to deploy this branch to see if it helps out the heroku environments.